tools/generate-helpers/pg-table-bindings/list.go

```diff
@@ -62,14 +62,16 @@ func init() {
 		&storage.Notifier{}:                                     resources.Integration,
 		&storage.PermissionSet{}:                                resources.Role,
 		&storage.Pod{}:                                          resources.Deployment,
-		&storage.PolicyCategory{}:                               resources.Policy,
-		&storage.ProcessBaselineResults{}:                       resources.DeploymentExtension,
-		&storage.ProcessBaseline{}:                              resources.DeploymentExtension,
-		&storage.ProcessIndicator{}:                             resources.DeploymentExtension,
-		&storage.ResourceCollection{}:                           resources.WorkflowAdministration,
-		&storage.ReportConfiguration{}:                          resources.VulnerabilityReports,
-		&storage.Risk{}:                                         resources.DeploymentExtension,
-		&storage.Role{}:                                         resources.Role,
+		// TODO: ROX-13888 Replace Policy with WorkflowAdministration.
```

Comment:
super-nit: You could also add `&storage.Policy{}: resource.Policy,` in this list to prepare the migration of the policy store as well.